portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/users/gdpr/GDPR.testcase

```diff
@@ -1150,8 +1150,6 @@ definition {
 		}
 
 		task ("Filter by name and assert that data exports are listed alphabetically") {
-			ManagementBar.toggleViewOrder(order = "list-down");
```

Comment:
Could you investigate why this is not needed in the test? Since the default order is list-up as we can tell on portal.

Reply:
done